cranelift/native/src/lib.rs

```diff
@@ -138,10 +138,15 @@ mod tests {
             let flag_builder = settings::builder();
             let isa = isa_builder.finish(settings::Flags::new(flag_builder));
             if cfg!(any(unix, target_os = "nebulet")) {
-                assert_eq!(isa.default_call_conv(), CallConv::SystemV);
+                if cfg!(target_os = "macos") {
```

Comment:
Maybe move this `if cfg!(` above the `if cfg!(any(unix, ...`?

Reply:
yep, and it's not even correct for macos x64, will fix.